### PR TITLE
Update `ilks2ascii.sh`

### PR DESCRIPTION
**Pull request by naszam**
- [x] update `ilks2ascii.sh` to apply `keys2ascii.sh` (dss-chain-log) improvements.
- [x] add script to Makefile
- [x] cleanups

### CODE REVIEW

File: scripts/ilks2ascii.sh [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+set -e
 
-for ilk in $(echo "$ILKS" | sed "s/,/ /g")
-do
-	seth --to-ascii "$ilk"
+[[ "$ETH_RPC_URL" ]] || { echo "Please set a ETH_RPC_URL"; exit 1; }
+
+CHANGELOG=0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F
+
+ILK_REGISTRY=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "ILK_REGISTRY")")")
```
Comment (brianmcmichael) [inline scripts/ilks2ascii.sh:11]: Can you update this script to use foundry?

Comment (brianmcmichael) [inline scripts/ilks2ascii.sh:11]: Getting that gas price error with seth here. If forge doesn't resolve the issue we should maybe see if we can file an issue with them to not rely on your local wallet balance for `call`s.

```
bmcmichael@bmcx1e:~/makerdao/ilk-registry$ ILK_REGISTRY=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "ILK_REGISTRY")")")
seth-rpc: {"id":1,"jsonrpc":"2.0","method":"eth_call","params":[{"data":"0x21f8a721494c4b5f52454749535452590000000000000000000000000000000000000000","from":"0xC0dE1AB51082442f821bA273273B4F7d32521D8b","gas":"0x1312d00","maxFeePerGas":"0x5d21dba000","maxPriorityFeePerGas":"0x77359400","to":"0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F"},"latest"]}
seth-rpc: error:   code       -32000
seth-rpc: error:   message    err: insufficient funds for gas * price + value: address 0xC0dE1AB51082442f821bA273273B4F7d32521D8b have 1691614669368968099 want 8000000000000000000 (supplied gas 20000000)
```

Comment (naszam) [reply scripts/ilks2ascii.sh:16]: I've addressed the seth gas price error, following previous [pattern](https://github.com/makerdao/spells-goerli/blob/master/scripts/wards.sh#L8) used in the `wards.sh` script in 5338bfb.

Comment (naszam) [reply scripts/ilks2ascii.sh:16]: I've tried replacing `seth` with `cast` but doesn't seem to work out-of-the-box, we could migrate to foundry in a separate pr eventually, to reproduce and address those issues.

**Review by brianmcmichael — APPROVED** [review]
LGTM

**Pull request merged**